packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapMarkerController.m

```diff
@@ -56,85 +53,55 @@ - (void)removeMarker {
   self.marker.map = nil;
 }
 
-- (void)setAlpha:(float)alpha {
-  self.marker.opacity = alpha;
-}
-
-- (void)setAnchor:(CGPoint)anchor {
-  self.marker.groundAnchor = anchor;
-}
-
-- (void)setDraggable:(BOOL)draggable {
-  self.marker.draggable = draggable;
-}
-
-- (void)setFlat:(BOOL)flat {
-  self.marker.flat = flat;
-}
-
-- (void)setIcon:(UIImage *)icon {
-  self.marker.icon = icon;
-}
-
-- (void)setInfoWindowAnchor:(CGPoint)anchor {
-  self.marker.infoWindowAnchor = anchor;
-}
-
-- (void)setInfoWindowTitle:(NSString *)title snippet:(NSString *)snippet {
-  self.marker.title = title;
-  self.marker.snippet = snippet;
-}
-
-- (void)setPosition:(CLLocationCoordinate2D)position {
-  self.marker.position = position;
-}
+- (void)updateFromPlatformMarker:(FGMPlatformMarker *)platformMarker
+                       registrar:(NSObject<FlutterPluginRegistrar> *)registrar
+                     screenScale:(CGFloat)screenScale {
+  self.clusterManagerIdentifier = platformMarker.clusterManagerId;
+  self.consumeTapEvents = platformMarker.consumeTapEvents;
 
-- (void)setRotation:(CLLocationDegrees)rotation {
-  self.marker.rotation = rotation;
-}
+  // Set the marker's user data with current identifiers.
+  FGMSetIdentifiersToMarkerUserData(self.markerIdentifier, self.clusterManagerIdentifier,
+                                    self.marker);
 
-- (void)setVisible:(BOOL)visible {
   // If marker belongs the cluster manager, visibility need to be controlled with the opacity
   // as the cluster manager controls when marker is on the map and when not.
```

Comment:
nit: Perhaps set a variable under this comment like `useOpacityForVisibility` and pass that in instead of passing `self.clusterManagerIdentifier` directly. Or could rename `usingOpacityForVisibility` to something like `belongsToClusterManager` and move the comment down to the if statement in `updateMarker`.  I had to go look at the original code to understand what this comment was about

Reply:
> Perhaps set a variable under this comment like `useOpacityForVisibility` and pass that in instead of passing `self.clusterManagerIdentifier` directly.

Good idea, done!